Druid-Tests/DruidTestRTLCompiler.class.st

```diff
@@ -3424,7 +3424,7 @@ DruidTestRTLCompiler >> gen_pushLiteralVariable16CasesBytecode [
 		      ifNone: [ ^ self unknownBytecode ].
 	live := live bitOr: (self registerMaskFor: t0).
 	self genMoveConstant: methodObj R: t0.
-	self MoveM64: 0 + LiteralStart << 3 + 8 r: t0 R: t0.
+	self MoveM64: 15 + LiteralStart << 3 + 8 r: t0 R: t0.
```

Comment:
This change is because all cases are tested using the same `gen` method, right? 🤔

Reply:
yup